docs/reference/components/sources/postgresql_metrics.cue

```diff
@@ -94,7 +95,8 @@ components: sources: postgresql_metrics: {
 		}
 		exclude_databases: {
 			description: """
-				A list of databases to match against the `datname` column for which you don't want to collect metrics from.
```

Comment:
It might be nice to add an example to the below examples which uses a regex. I'd suggest `template.*` since it is likely that users will often want to exclude those databases.